src/components/timeline/RowDragHandleHeight.svelte

```diff
@@ -5,6 +5,7 @@
   import { ViewConstants } from '../../enums/view';
 
   export let rowHeight: number = 0;
+  export let largeRow: boolean = false; // if the row contains an activity and external event layer, in compacted mode
```

Comment:
Discuss if this is the right approach for combined activity + event row sizing

Reply:
Per meeting: we will stick to one drawn row on collapse of an event/activity row, making these lines entirely obsolete. This is implemented in [change collapsed row behavior to one row](https://github.com/NASA-AMMOS/aerie-ui/commit/1a28aaed074d46a57432497ee5f7a5adb9a0f042).